automated install/basic-install.sh

```diff
@@ -444,6 +444,7 @@ setDNS() {
       Level3 ""
       Norton ""
       Comodo ""
+      DNS.WATCH ""
```

Comment:
`CamelCase` please, just to match with the rest of the options. So `DNSWatch`, same on line 478. Thanks!